src/code/qualification/runQualificationWorkflow.m

```diff
@@ -129,21 +129,21 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
 
 %---------------------------------------------------
 % Plot GOFMerged
-for i=1:length(TaskList)
-    if strcmp(TaskList{i}, 'GOFMergedPlots')
+for TaskListIndex=1:length(TaskList)
+    if strcmp(TaskList{TaskListIndex}, 'GOFMergedPlots')
         
-        for j=1:length(ConfigurationPlan.Plots.GOFMergedPlots)
+        for GOFMergedIndex=1:length(ConfigurationPlan.Plots.GOFMergedPlots)
             % Check if misread of GOFMergedPlots as cells
-            if iscell(ConfigurationPlan.Plots.GOFMergedPlots(j))
-                GOFMerged=ConfigurationPlan.Plots.GOFMergedPlots{j};
+            if iscell(ConfigurationPlan.Plots.GOFMergedPlots(GOFMergedIndex))
+                GOFMerged=ConfigurationPlan.Plots.GOFMergedPlots{GOFMergedIndex};
             else
-                GOFMerged=ConfigurationPlan.Plots.GOFMergedPlots(j);
+                GOFMerged=ConfigurationPlan.Plots.GOFMergedPlots(GOFMergedIndex);
             end
             
-            for k=1:length(GOFMerged)
+            for GOFMergedGroupIndex=1:length(GOFMerged)

```

Comment:
This for loop `for GOFMergedGroupIndex=1:length(GOFMerged.Groups) ` seems to be useless??
`length(GOFMerged)` is always 1 and here (in **runQualificationWorkflow**) there is no need to loop through the Groups anyway? (loop through the Groups is done in **plotQualificationGOFMerged**)
Or do I miss something?

Reply:
You are completely right,
I just removed the loop.